api/models/DownloadStatistic.js

```diff
@@ -37,7 +45,12 @@ module.exports = {
 
 
   options: {
-    underscored: true
+    underscored: true,
+    classMethods: {
+      getMonthlySplittedDownloads :function(package_name){
+        return sequelize.query("SELECT SUM(indirect_downloads) AS indirect_downloads,SUM(direct_downloads) AS direct_downloads FROM DownloadStatistics WHERE date >= current_date() - interval '30' day and package_name = :package",{ replacements: { package: package_name }, type: sequelize.QueryTypes.SELECT });
```

Comment:
Would be better if this use `- interval 1 month` instead of `- 30 day` for month with a number of days different than 30